### PR TITLE
Added an example CloudLayer render texture + rotation for cloud layer

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
The PR adds an example render texture for the CloudLayer override and a parameter to rotate the cloud layer.

Also removes the shader warnings about uninitiliazed variables

### CODE REVIEW

**Comment by iM0ve** [comment]
Testing status: **In Progress**

**TL: DR No technical issues, but a few UX ones.**

1) **Adding the cloud override does nothing out of the box**. We should aim for good default values when adding an override, then users can find their way from there. Its difficult to find that we already have a nice cloud texture ready to be used.

**Now**
N9JoAn0pQS

**Expected**
d9E7Fjnam0

Possible solutions:
* Add the texture and enable scrolling parameters by default
* Or/and add the clouds to the New scene template. To increase discoverability.

2) This is a more minor issue. **The advanced cloud opacity settings are hard to discover.** 

image

Possible solutions:
* Add to documentation

**Comment by adrien-de-tocqueville** [comment]
> 2) The advanced cloud opacity settings are hard to discover.
Fixed by adding the material inspector in the CloudLayer inspector

> 4) parameter on the DefaultClouds material with a tooltip
its removed

> 5) Physically based sky is vastly different from HDRP or Gradient
Hacky fix but it should be similar now

File: com.unity.render-pipelines.high-definition/Documentation~/Override-Cloud-Layer.md [hunk 1]
```diff
@@ -11,13 +11,19 @@ The **Cloud Layer** uses the [Volume](Volumes.html) framework, so to enable and
 
 After you add a **Cloud Layer** override, you must enable it in the override itself. In the override, Check the **Enable** property. HDRP now renders **Cloud Layer** for any Camera this Volume affects.
 
+<a name="DefaultCloudMap"></a>
+
+## Using the default Cloud Map
+
+An example Cloud Map texture named `DefaultCloudLayer` is included in the HDRP package.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Override-Cloud-Layer.md:18]: HDRP includes an example Cloud Map texture named **DefaultCloudLayer**.

File: com.unity.render-pipelines.high-definition/Documentation~/Override-Cloud-Layer.md [hunk 2]
```diff
@@ -11,13 +11,19 @@ The **Cloud Layer** uses the [Volume](Volumes.html) framework, so to enable and
...
+As it is a CustomRenderTexture, you can duplicate the asset in your project and tweak some parameters. To do so, first duplicate the `DefaultCloudLayer` CustomRenderTexture. Then duplicate the `DefaultCloudLayerMaterial` and select it in the inspector of your CustomRenderTexture.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Override-Cloud-Layer.md:19]: This example Cloud Map is a read-only CustomRenderTexture. This means that, if you want to modify any of its properties, you need to:
1. Duplicate the **DefaultCloudLayer** asset.
2. Duplicate the Material the **DefaultCloudLayer** asset references. This Material is called **DefaultCloudLayerMaterial**.
3. In the Inspector for the new **DefaultCloudLayer**, assign the new **DefaultCloudLayerMaterial** to the **Material** property.
